src/components/dropdown-menu/DropdownMenu.js

```diff
@@ -47,25 +47,25 @@ class DropdownMenu extends React.Component<Props, State> {
     componentDidUpdate(prevProps: Props, prevState: State) {
         if (!prevState.isOpen && this.state.isOpen) {
             // When menu is being opened
-            document.addEventListener('click', this.handleDocumentClick, true);
-            document.addEventListener('contextmenu', this.handleDocumentClick, true);
+            document.addEventListener('click', this.handleDocumentClick, false);
+            document.addEventListener('contextmenu', this.handleDocumentClick, false);
```

Comment:
This needs to be tested pretty extensively in consuming apps to be sure this is playing nicely.

Reply:
agree
Created PR https://jira.inside-box.net/browse/MONEY-5514

Reply:
add shouldCloseOnEventBubbling prop that if set to true, will close the dropdown only if event bubbles up instead of capturing the event in document.

Reply:
rename the prop to useBubble and rename the corresponding pop in Tooltip.tsx as stopBubble